config/locales/en.yml

```diff
@@ -212,7 +212,7 @@ en:
         job_summary: >-
           Give a brief summary of the job that job seekers can quickly scan.
           This will be the first thing they see on the listing or in search engine results
-        about_school: "This will appear next to the job details under 'School overview'. Feel free to adapt this text"
+        about_organisation: "This will appear next to the job details under 'School overview'. Feel free to adapt this text"
```

Comment:
`This will appear next to the job details under 'School overview'` this part may need to change as well to consider the case of a trust

Reply:
This and the error one you can have `about_school` and `about_trust` and use a different one, or something more complicated would be passing `count: current_organisation.is_a?(School) ? 0 : 1` and adding `zero` and `one` translations under `about_organisation`